ubuntu/ubuntu-20.04.json

```diff
@@ -1,44 +1,62 @@
 {
+  "variables": {
+    "SSH_PUB_KEY": "",
+    "SSH_PRIVATE_KEY_FILE":  ""
+  },
   "builders": [
     {
       "boot_command": [
-	"<esc><esc><f6><esc><wait>",
+        "<enter><enter><f6><esc><wait>",
+        "<bs><bs><bs><bs><bs><bs><bs><bs>",
```

Comment:
What are all those `<bs>` for?

Reply:
the previous keystrokes jump to the boot command line editing, those `<bs>` remove the original command line, so we can then put ours. 

This was discovered after testing interactively, basically the existing command line in this ubuntu version was not playing nicely with the rest of the options, so we remove it completely and add the one that we want with
```
	"/casper/vmlinuz ",
        "initrd=/casper/initrd ",
```
and then the rest of the options